lib/broccoli/default-packager.js

```diff
@@ -833,30 +834,42 @@ module.exports = class DefaultPackager {
 
         let destDir,
             testSrcTree,
-            srcDir;
+            srcDir,
+            dirNotFound;
 
         // ember-addon
         if (this.name === 'dummy') {
           testSrcTree = 'src';
           destDir = `${this.project.name()}/src`;
+
+          // Classic Addon with MU dummy app `kitchen-sink-with-mu-dummy-app` will throw: `Dir not found: src` exception
```

Comment:
Is there any better way to avoid this error? `Funnel.allowEmpty` does not solve the issue.